source/blog/2017-08-07-savadoro-beta-available.html.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: Savadoro Beta Available
+date: 2017-08-07 05:00 UTC
+author: Gary A. Newsome
+category: products
+tags: savadoro, timer, products
+---
+
+SavvySoftWorks, LLC is proud to announce the Savadoro timer.
+
+Savadoro is Pomodoro Technique timer, while this is still in a very early beta stage, we have big ideas for the timer long term. Including integration into Slack as well as the ability to track you and your teams past progress.
+
+For a little background, per [https://en.wikipedia.org/wiki/Pomodoro_Technique](Wikipedia) the Pomodoro Technique is a time management technique developed in the late 1980s. Basically, you decide upon a task, then set the timer for 25 minutes. After the timer expires you take a short 3-5 minute break, and then repeat. after 4 times through the cycle you take a longer 15-20 minute break.
+
+This technique helps you stay on task as it creates specific periods of focus on a designated task. It also helps you avoid the perverbial "rabbit hole" of getting lost in a specific problem for too long, as the timer will interupt you at the designated time. Giving you a "brain break" or allowing you a pause to seek help.
+
+The timer itself is free for you to try, and is available for download at out Github page. [https://github.com/SavvySoftWorksLLC/savadoro_electron/releases](https://github.com/SavvySoftWorksLLC/savadoro_electron/releases)
```

Comment:
Point to the current release and publish the release.
Also we should resolve https://github.com/SavvySoftWorksLLC/savadoro_electron/pull/28 & https://github.com/SavvySoftWorksLLC/savadoro_electron/pull/29 first since they help with the icon and platform distribution issues